### PR TITLE
Feature/add issue

### PR DESCRIPTION
**Pull request by andrewRv**

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -270,7 +270,7 @@ contextmanager-decorators=contextlib.contextmanager
 # List of members which are set dynamically and missed by pylint inference
 # system, and so shouldn't trigger E1101 when accessed. Python regular
 # expressions are accepted.
-generated-members=db.*
+generated-members=db.*, Issues.objects
```
Comment (NewIvanIvanov) [inline .pylintrc]: change to
generated-members=db.*, Issues.objects, Category.objects

File: client/apps/city_issues/forms/forms.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+from city_issues.models.issues import Issues, Category
+
+
+class IssueForm(forms.ModelForm):
+    class Meta:
+        model = Issues
+        fields = ['description', 'category', 'location_lat', 'location_lon', 'name']
+
+    name = forms.CharField(
+        max_length=350,
```
Comment (NewIvanIvanov) [inline client/apps/city_issues/forms/forms.py]: IMHO so long name not needed, may be change to max_length=50

Comment (andrewRv) [reply client/apps/city_issues/forms/forms.py]: I agree, maybe typos

File: client/basic/urls.py [hunk 3]
```diff
@@ -26,4 +26,5 @@
     url(r'^map/getissuebyid/(?P<issue_id>[0-9]+)$',
         get_issue_data, name='issue_data'),
     url(r'^map/getissuesall/$', get_all_issues_data, name='all_issues_data'),
+    url(r'^add-issue', IssueCreate.as_view(), name='create_issue'),
```
Comment (KostSvitlana) [inline client/basic/urls.py]: maybe url without hyphen like `addissue`?

**Review by KostSvitlana — APPROVED** [review]

**Review by alevitsky — APPROVED** [review]

**Pull request merged**